runtime/src/bank.rs

```diff
@@ -382,6 +382,13 @@ impl Bank {
     pub fn last_blockhash(&self) -> Hash {
         self.blockhash_queue.read().unwrap().last_hash()
     }
+    pub fn get_confirmed_blockhash(&self) -> Hash {
```

Comment:
nit: maybe call this `confirmed_last_blockhash()` for consistency with `transaction_count()`/`confirmed_transaction_count()`